docs/Getting-started.md

```diff
@@ -68,3 +61,6 @@ And to execute the tests in multisite, run:
 ```
 npm run test-php-multisite
 ```
+
+## Other ways to contribute
+Not a developer? You can still help! We’re always looking for people to assist with support on our WordPress.org support forum. Please reach out in the #performance Slack channel for more details.
```

Comment:
Now that we're adding this section, I think it would be good to also call out the possibility to help translate. For both places, it would be good to include links to the support forums and translation projects:
* https://wordpress.org/support/plugin/performance-lab/
* https://translate.wordpress.org/projects/wp-plugins/performance-lab/